app/controllers/kondos_controller.rb

```diff
@@ -51,8 +51,7 @@ def reserve
     authorize @kondo
     @kondo.reserved = !@kondo.reserved
     @kondo.save!
-    redirect_to dashboard_index_path
-
+    redirect_to dashboard_path
```

Comment:
the routes are telling me dashboard_index_path but thats not working I saw that yesterday in another situation too
why?

Reply:
I changed it to show, because that was the easiest most straightforward way to fix it :) Also because we show 1 dashboard, not an index of dashboards